src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[3];
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure defaultFigure = figureSupplier.getDefaultFigure();
+        figures[0] = defaultFigure;
+        defaultFigure.draw();
+        for (int i = 1; i < figures.length; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+            figures[i].draw();
+        }
+    }
```

Comment:
refactor this code, so you initialize your array in one loop and then in other for-each loop you draw all figures